### PR TITLE
Iss33

### PR DESCRIPTION
**Pull request by sahilambre**
Issue #33  Update documentation

### CODE REVIEW

**Review by tanmayc07 — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -1,4 +1,38 @@
 # A Blogging website built with Nodejs
```
[the remark this answers is not in the document]
Comment (sahilambre) [reply README.md:1]: Ok, even I was thinking of the same.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,38 @@
...
+
+2. Download all the **packages** and it's **dependencies**. Open the terminal and type
+
+```bash
+npm install
+```
+
+3. Create `.env` file and insert proper **key values** (refer `.env.sample`).
+
+4. To **start** the server run
```
[the remark this answers is not in the document]
Comment (sahilambre) [reply README.md:29]: ok

**Comment by sahilambre** [comment]
Changes have been made. Please check @tanmayc07

**Comment by tanmayc07** [comment]
I think we should add the header content as body after header, and write a new header. Something like Think it Write it!

**Comment by sahilambre** [comment]
> 
> 
> I think we should add the header content as body after header, and write a new header. Something like Think it. Write it!

Something like this??
image

**Comment by tanmayc07** [comment]
No I meant just change the text in header as `Think it. Write it!` 
And below that write in that previous header text as normal text.

**Comment by tanmayc07** [comment]
@sahilambre Are you working on this?

**Comment by sahilambre** [comment]
Yes

File: README.md [hunk 3]
```diff
@@ -1,4 +1,41 @@
...
+  <img alt="Github status" src="https://img.shields.io/badge/license-MIT-brightgreen">
+</p>
+<p>
+  <h1 align="center">Think it. Write it</h1>
+ <p align="center">A Blogging website built with Nodejs 🖋️</p>
+</p>
+
+
+
+## **How to run code locally**
```
Comment (tanmayc07) [inline README.md:20]: Change it to `###` . It still looks equal as h1

**Review by tanmayc07 — CHANGES_REQUESTED** [review]

**Review by tanmayc07 — APPROVED** [review]

**Pull request merged**